sim/SPI_v3/components/SPILoopbackComposite.py

```diff
@@ -0,0 +1,39 @@
+'''
+==========================================================================
+SPILoopbackComposite.py
+==========================================================================
+A composition module consisting of SPIMinionAdapterComposite and Loopback modules.
+For use with testing SPI communication
+
+Author : Kyle Infantino
+  Date : Jan 30, 2022
+
+'''
+
+from pymtl3 import *
+from .SPIMinionAdapterCompositePRTL import SPIMinionAdapterCompositePRTL
+from .Loopback import Loopback
+
+class SPILoopbackComposite( Component ):
+
+    def construct( s, nbits=32 ):
```

Comment:
Same thing for this file. Can we change it to 2-space indention?

Reply:
Indentation changed to 2-space